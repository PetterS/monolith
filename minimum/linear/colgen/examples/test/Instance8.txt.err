Number of days: 28
-- Shift E, 480 min.
-- Shift D, 480 min.
-- Shift L, 480 min.
-- Shift N, 480 min.
Number of shifts: 4
-- Shift 0. OK after = [0, 1, 2, 3] forbidden_after = []
-- Shift 1. OK after = [1, 2, 3] forbidden_after = [0]
-- Shift 2. OK after = [2, 3] forbidden_after = [0, 1]
-- Shift 3. OK after = [3] forbidden_after = [0, 1, 2]
-- Staff  A: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 0, 4].
-- Staff  B: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  C: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 0, 4].
-- Staff  D: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 14, 0].
-- Staff  E: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  F: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  G: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 0, 4].
-- Staff  H: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 0].
-- Staff  I: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  J: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 14, 4].
-- Staff  K: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  L: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  M: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 14, 4].
-- Staff  N: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 14, 4].
-- Staff  O: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 14, 0].
-- Staff  P: Consequtive limit [2, 5]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 0].
-- Staff  Q: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 14, 4].
-- Staff  R: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 0, 0].
-- Staff  S: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [0, 28, 14, 0].
-- Staff  T: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 14, 0].
-- Staff  U: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  V: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 14, 4].
-- Staff  W: Consequtive limit [2, 6]. Time limit [8160, 8640]. Max weekends: 2. Max shifts: [14, 28, 0, 4].
-- Staff  X: Consequtive limit [1, 5]. Time limit [4680, 5160]. Max weekends: 3. Max shifts: [0, 28, 0, 2].
-- Staff  Y: Consequtive limit [1, 5]. Time limit [4680, 5160]. Max weekends: 3. Max shifts: [0, 28, 8, 0].
-- Staff  Z: Consequtive limit [1, 5]. Time limit [4680, 5160]. Max weekends: 3. Max shifts: [8, 28, 8, 2].
-- Staff AA: Consequtive limit [1, 5]. Time limit [4680, 5160]. Max weekends: 3. Max shifts: [0, 28, 8, 2].
-- Staff AB: Consequtive limit [1, 5]. Time limit [4680, 5160]. Max weekends: 3. Max shifts: [8, 28, 8, 2].
-- Staff AC: Consequtive limit [1, 5]. Time limit [2940, 3420]. Max weekends: 3. Max shifts: [0, 28, 0, 1].
-- Staff AD: Consequtive limit [1, 5]. Time limit [2940, 3420]. Max weekends: 3. Max shifts: [0, 28, 5, 1].
Number of staff: 30
Read 139 shift on requests.
Read 86 shift off requests.
Read 112 cover constraints.
Dividing all minutes by 480
Setting up colgen problem...[  OK  ]
Creating initial rosters...[  OK  ]
-- Objective for initial columns: 15080
  Iter |       Objective      |    Problem   |    Pool   | Fixed  |                  Time                  | Frac. |
       |    frac.     rounded |   size  diff | size gen. |        |    fix       gen     solve   tot. cum. |       |
      1      15080      15080        0  254     254    0      0        n/a       n/a     n/a     n/a          0%
      2    8325.78       8769       56   56     310   56      0        n/a       n/a     n/a     n/a          3%
      3    6400.98       7667      101   45     360   50      0        n/a       n/a     n/a     n/a          0%
      4    5288.67       6958      138   37     403   43      0        n/a       n/a     n/a     n/a          9%
      5    4818.14       6449      178   40     457   54      0        n/a       n/a     n/a     n/a          5%
      6    4269.03       6561      227   49     515   58      0        n/a       n/a     n/a     n/a          1%
      7    3973.57       5561      243   16     563   48      0        n/a       n/a     n/a     n/a          8%
      8    3837.23       6270      267   24     619   56      0        n/a       n/a     n/a     n/a          7%
      9     3648.4       5766      279   12     665   46      0        n/a       n/a     n/a     n/a          8%
     10    3523.46       5758      281    2     713   48      0        n/a       n/a     n/a     n/a          9%
     11    3399.63       5866      278   -3     767   54      0        n/a       n/a     n/a     n/a          2%
     12    3292.65       5674      273   -5     812   45      0        n/a       n/a     n/a     n/a          2%
     13    3218.01       5877      268   -5     858   46      0        n/a       n/a     n/a     n/a          3%
     14     3148.7       6073      256  -12     904   46      0        n/a       n/a     n/a     n/a          4%
     15    3097.94       6178      256    0     954   50      0        n/a       n/a     n/a     n/a          5%
     16    3021.36       6283      237  -19     996   42      0        n/a       n/a     n/a     n/a          9%
     17    2943.34       6489      263   26    1048   52      0        n/a       n/a     n/a     n/a          6%
     18    2887.37       6178      252  -11    1099   51      0        n/a       n/a     n/a     n/a          6%
     19     2854.6       6080      257    5    1144   45      0        n/a       n/a     n/a     n/a          5%
     20    2796.08       5688      273   16    1184   40      0        n/a       n/a     n/a     n/a          5%
     21     2766.3       5991      275    2    1230   46      0        n/a       n/a     n/a     n/a          5%
     22    2721.44       5989      247  -28    1276   46      0        n/a       n/a     n/a     n/a          0%
     23    2685.64       5690      234  -13    1320   44      0        n/a       n/a     n/a     n/a          1%
     24    2672.35       5177      222  -12    1370   50      0        n/a       n/a     n/a     n/a          2%
     25    2664.94       5990      217   -5    1409   39      0        n/a       n/a     n/a     n/a          5%
     26    2657.77       5888      213   -4    1449   40      0        n/a       n/a     n/a     n/a          6%
     27    2652.21       5885      203  -10    1498   49      0        n/a       n/a     n/a     n/a          9%
     28    2647.25       5989      199   -4    1532   34      0        n/a       n/a     n/a     n/a          0%
     29    2642.36       5373      196   -3    1580   48      0        n/a       n/a     n/a     n/a          0%
     30    2628.56       5476      185  -11    1625   45      0        n/a       n/a     n/a     n/a          1%
     31    2621.31       5277      188    3    1662   37      0        n/a       n/a     n/a     n/a          0%
     32    2582.11       6375      186   -2    1704   42      0        n/a       n/a     n/a     n/a          1%
     33    2567.56       5366      201   15    1756   52      0        n/a       n/a     n/a     n/a          7%
     34    2558.28       4958      204    3    1793   37      0        n/a       n/a     n/a     n/a          8%
     35    2548.36       5059      199   -5    1839   46      0        n/a       n/a     n/a     n/a          7%
     36    2543.47       4958      240   41    1880   41      0        n/a       n/a     n/a     n/a          1%
     37    2535.04       5167      221  -19    1921   41      0        n/a       n/a     n/a     n/a          2%
     38    2523.13       5269      215   -6    1963   42      0        n/a       n/a     n/a     n/a          5%
     39     2503.9       4552      216    1    2002   39      0        n/a       n/a     n/a     n/a          6%
     40    2499.69       4751      227   11    2052   50      0        n/a       n/a     n/a     n/a          4%
     41    2495.65       4954      214  -13    2097   45      0        n/a       n/a     n/a     n/a          5%
     42    2494.47       4552      200  -14    2142   45      0        n/a       n/a     n/a     n/a          7%
     43    2491.42       4751      201    1    2187   45      0        n/a       n/a     n/a     n/a          8%
     44     2473.6       4751      206    5    2232   45      0        n/a       n/a     n/a     n/a          6%
     45    2465.65       5053      220   14    2279   47      0        n/a       n/a     n/a     n/a          5%
     46    2460.63       4953      202  -18    2320   41      0        n/a       n/a     n/a     n/a          7%
     47    2456.58       4953      211    9    2356   36      0        n/a       n/a     n/a     n/a          6%
     48    2455.18       4751      201  -10    2403   47      0        n/a       n/a     n/a     n/a          9%
     49    2451.23       4853      181  -20    2438   35      0        n/a       n/a     n/a     n/a          4%
     50    2447.18       4749      176   -5    2479   41      0        n/a       n/a     n/a     n/a          7%
     51    2444.46       5054      181    5    2510   31      0        n/a       n/a     n/a     n/a          5%
     52    2443.27       5359      183    2    2542   32      0        n/a       n/a     n/a     n/a          6%
     53    2442.74       5359      239   56    2570   28      0        n/a       n/a     n/a     n/a          5%
-- Fixed for [A: 13, B: 11, C: 13, D: 8, E: 2, F: 18, G: 10, H: 18, I: 14, J: 8, K: 16, L: 17, M: 12, N: 7, O: 15, P: 7, Q: 18, R: 17, S: 17, T: 14, U: 17, V: 17, W: 8, X: 7, Y: 5, Z: 4, AA: 2, AB: 3, AC: 4, AD: 4] at level 0.75.
-- Fixed to zero for [A: 91, B: 68, C: 80, D: 76, E: 80, F: 67, G: 77, H: 83, I: 54, J: 73, K: 66, L: 95, M: 62, N: 60, O: 75, P: 77, Q: 64, R: 88, S: 84, T: 56, U: 61, V: 74, W: 63, X: 90, Y: 83, Z: 77, AA: 75, AB: 66, AC: 96, AD: 91] at level 0.05.
-- Members remaining: (A, 2) (D, 2) (E, 2) (G, 2) (J, 2) (M, 2) (N, 3) (P, 2) (T, 2) (W, 2) (X, 2) (Y, 2) (Z, 4) (AA, 5) (AB, 2) (AC, 4) (AD, 2) and total is 17.
     54    3510.26       4552       71 -168     340   18   1639        n/a       n/a     n/a     n/a          0%
     55    3076.62       3953      108   37     361   21      0        n/a       n/a     n/a     n/a          2%
     56    2857.56       4051      122   14     379   18      0        n/a       n/a     n/a     n/a          9%
     57    2741.14       3854      139   17     396   17      0        n/a       n/a     n/a     n/a          3%
     58    2699.11       3547      157   18     410   14      0        n/a       n/a     n/a     n/a          1%
     59    2668.83       4056      160    3     424   14      0        n/a       n/a     n/a     n/a          4%
     60    2635.41       3651      164    4     438   14      0        n/a       n/a     n/a     n/a          7%
     61    2632.79       3651      175   11     451   13      0        n/a       n/a     n/a     n/a          6%
     62    2629.29       3854      172   -3     465   14      0        n/a       n/a     n/a     n/a          8%
     63    2627.74       3854      171   -1     478   13      0        n/a       n/a     n/a     n/a          0%
     64    2612.39       3748      169   -2     487    9      0        n/a       n/a     n/a     n/a          6%
     65    2611.29       3943      171    2     497   10      0        n/a       n/a     n/a     n/a          6%
     66    2606.44       3947      162   -9     509   12      0        n/a       n/a     n/a     n/a          7%
     67    2603.66       3848      172   10     521   12      0        n/a       n/a     n/a     n/a          7%
     68     2601.5       4047      162  -10     532   11      0        n/a       n/a     n/a     n/a          4%
     69     2600.8       4244      166    4     539    7      0        n/a       n/a     n/a     n/a          7%
-- Fixed for [A: 4, B: 6, C: 3, D: 5, E: 1, G: 4, I: 4, J: 8, K: 1, N: 3, O: 3, P: 1, T: 1, W: 5, Y: 1, AA: 3, AB: 3, AC: 2, AD: 1] at level 0.75.
-- Fixed to zero for [B: 6, C: 3, D: 9, F: 10, G: 3, H: 10, I: 14, J: 11, K: 15, M: 14, N: 10, O: 12, P: 7, Q: 12, S: 5, T: 19, U: 15, V: 17, W: 5, X: 5, Y: 7, Z: 9, AA: 13, AB: 11, AC: 3, AD: 5] at level 0.05.
-- Members remaining: (C, 2) (D, 3) (E, 5) (G, 4) (K, 2) (M, 5) (N, 3) (P, 6) (S, 2) (T, 2) (W, 2) (X, 4) (Y, 4) (Z, 7) (AA, 2) (AB, 2) (AC, 3) (AD, 2) and total is 18.
     70    2704.38       3136      105  -61     366    9    128        n/a       n/a     n/a     n/a          5%
     71    2639.48       3336      124   19     379   13      0        n/a       n/a     n/a     n/a          8%
     72     2611.8       3846      125    1     389   10      0        n/a       n/a     n/a     n/a          1%
     73    2601.03       3544      128    3     398    9      0        n/a       n/a     n/a     n/a          6%
     74     2585.7       3747      136    8     406    8      0        n/a       n/a     n/a     n/a          4%
     75     2585.5       3543      141    5     410    4      0        n/a       n/a     n/a     n/a          2%
-- Fixed for [C: 1, D: 2, E: 1, G: 3, J: 1, Z: 1, AA: 5] at level 0.75.
-- Fixed to zero for [B: 5, C: 2, F: 5, G: 1, H: 1, I: 1, J: 1, K: 1, M: 1, N: 7, Q: 1, R: 1, S: 3, T: 1, U: 6, V: 4, X: 1, Y: 1, AB: 3, AD: 2] at level 0.05.
-- Members remaining: (D, 2) (E, 2) (G, 2) (K, 2) (M, 3) (N, 4) (P, 6) (T, 2) (W, 2) (X, 3) (Y, 4) (Z, 5) (AB, 5) (AC, 3) (AD, 3) and total is 15.
     76       2587       3853      104  -37     371    2     38        n/a       n/a     n/a     n/a          5%
     77       2587       3851      119   15     378    7      0        n/a       n/a     n/a     n/a          3%
-- Fixed for [P: 1, W: 1, Y: 2, AB: 1, AD: 1] at level 0.75.
-- Fixed to zero for [A: 3, B: 5, G: 2, I: 1, K: 1, M: 1, R: 1, S: 2, T: 1, U: 1, W: 1, Z: 1, AD: 1] at level 0.05.
-- Members remaining: (D, 2) (E, 2) (G, 2) (K, 2) (M, 2) (N, 3) (P, 2) (T, 2) (W, 2) (X, 2) (Y, 3) (Z, 2) (AB, 2) (AC, 2) (AD, 3) and total is 15.
     78       2638       3851       85  -34     353    4     29        n/a       n/a     n/a     n/a          6%
     79       2638       3851       95   10     360    7      0        n/a       n/a     n/a     n/a          4%
-- Fixed to zero for [A: 1, B: 4, C: 7, F: 2, G: 2, I: 1, M: 2, N: 1, T: 5, U: 1, W: 3, Y: 1, AA: 1, AD: 1] at level 0.05.
-- Fixing column 485 to 1 for member N, was 0.5
-- Members remaining: (D, 2) (E, 2) (G, 2) (K, 2) (M, 2) (N, 2) (P, 2) (T, 2) (W, 2) (X, 2) (Y, 2) (Z, 2) (AB, 2) (AC, 2) (AD, 2) and total is 15.
     80    2837.67       3340       91   -4     365    5      1        n/a       n/a     n/a     n/a          3%
     81    2769.33       3037       83   -8     369    4      0        n/a       n/a     n/a     n/a          8%
     82       2760       2936       97   14     372    3      0        n/a       n/a     n/a     n/a          8%
     83       2734       2734       80  -17     375    3      0        n/a       n/a     n/a     n/a          0%
     84       2734       2734       87    7     378    3      0        n/a       n/a     n/a     n/a          0%
-- Stopping. Objective change is 0 with an integer feasible solution.
Colgen done.
Elapsed time : n/a.
  Iter |       Objective      |    Problem   |    Pool   | Fixed  |                  Time                  | Frac. |
       |    frac.     rounded |   size  diff | size gen. |        |    fix       gen     solve   tot. cum. |       |
      1       2638       3952       87    0    2878    0      0        n/a       n/a     n/a     n/a          6%
      2    2410.46       3648      427  340    2926   48      0        n/a       n/a     n/a     n/a          6%
      3     2124.3       3854      960  533    2978   52      0        n/a       n/a     n/a     n/a          1%
      4    2085.87       4363     1022   62    3031   53      0        n/a       n/a     n/a     n/a          2%
      5    2080.36       4061     1042   20    3079   48      0        n/a       n/a     n/a     n/a          1%
      6    2079.62       3855     1027  -15    3119   40      0        n/a       n/a     n/a     n/a          2%
-- Fixed for [A: 13, B: 7, C: 17, D: 14, E: 3, F: 18, G: 18, H: 15, I: 5, J: 12, K: 3, L: 17, M: 16, N: 15, O: 9, P: 8, Q: 10, R: 12, S: 18, T: 15, U: 17, V: 17, W: 12, X: 9, Y: 5, Z: 7, AA: 5, AB: 4, AC: 6, AD: 6] at level 0.75.
-- Members remaining: (A, 2) (B, 2) (D, 2) (E, 4) (H, 2) (I, 2) (J, 2) (K, 2) (M, 2) (O, 3) (P, 5) (Q, 2) (R, 2) (T, 2) (W, 3) (X, 3) (Y, 4) (Z, 2) (AA, 2) (AB, 5) (AC, 3) (AD, 2) and total is 22.
      7    2431.97       3252      164 -863     498   16   2637        n/a       n/a     n/a     n/a          6%
      8     2370.8       3255      149  -15     516   18      0        n/a       n/a     n/a     n/a          4%
      9    2326.45       3459      167   18     531   15      0        n/a       n/a     n/a     n/a          4%
     10    2305.52       3553      177   10     544   13      0        n/a       n/a     n/a     n/a          1%
     11    2299.21       3448      184    7     557   13      0        n/a       n/a     n/a     n/a          3%
     12    2286.65       3651      187    3     570   13      0        n/a       n/a     n/a     n/a          9%
     13    2284.49       3249      182   -5     581   11      0        n/a       n/a     n/a     n/a          1%
     14    2274.35       3545      170  -12     592   11      0        n/a       n/a     n/a     n/a          2%
     15    2265.91       3547      173    3     602   10      0        n/a       n/a     n/a     n/a          3%
     16    2258.82       3141      173    0     612   10      0        n/a       n/a     n/a     n/a          1%
     17    2251.57       3043      176    3     624   12      0        n/a       n/a     n/a     n/a          2%
     18    2247.78       3244      166  -10     632    8      0        n/a       n/a     n/a     n/a          3%
     19    2246.99       3244      175    9     640    8      0        n/a       n/a     n/a     n/a          0%
-- Fixed for [A: 4, B: 5, D: 1, E: 11, H: 2, J: 3, K: 15, M: 2, N: 2, O: 3, P: 10, Q: 1, R: 4, T: 2, W: 4, AA: 1] at level 0.75.
-- Members remaining: (B, 2) (D, 2) (H, 3) (I, 2) (J, 2) (O, 3) (Q, 5) (R, 2) (W, 2) (X, 2) (Y, 8) (Z, 6) (AA, 3) (AB, 5) (AC, 2) (AD, 2) and total is 16.
     20    2528.05       3443      104  -71     439    9    210        n/a       n/a     n/a     n/a          1%
     21    2391.81       3547      152   48     450   11      0        n/a       n/a     n/a     n/a          1%
     22    2388.47       3346      124  -28     461   11      0        n/a       n/a     n/a     n/a          6%
     23     2385.4       3549      138   14     470    9      0        n/a       n/a     n/a     n/a          4%
     24     2384.9       3144      139    1     481   11      0        n/a       n/a     n/a     n/a          5%
-- Fixed for [B: 1, J: 2, Q: 2, R: 1, X: 1, Z: 2, AB: 2] at level 0.75.
-- Members remaining: (B, 2) (D, 3) (E, 4) (H, 3) (I, 2) (J, 2) (O, 2) (Q, 2) (W, 2) (Y, 5) (Z, 2) (AA, 2) (AB, 2) (AC, 2) (AD, 4) and total is 15.
     25    2407.36       3548      105  -34     413    8     76        n/a       n/a     n/a     n/a          3%
     26    2396.88       3549      114    9     421    8      0        n/a       n/a     n/a     n/a          6%
     27    2384.25       3246      117    3     429    8      0        n/a       n/a     n/a     n/a          4%
     28       2384       3246      106  -11     434    5      0        n/a       n/a     n/a     n/a          4%
-- Fixed for [AD: 1] at level 0.75.
-- Members remaining: (B, 2) (D, 2) (E, 2) (H, 2) (I, 2) (J, 2) (O, 2) (Q, 2) (R, 2) (W, 2) (Y, 4) (Z, 2) (AA, 2) (AB, 2) (AC, 2) and total is 15.
     29       2384       3549      108    2     423    3     14        n/a       n/a     n/a     n/a          2%
-- Fixed for [Y: 1] at level 0.75.
-- Members remaining: (B, 2) (D, 2) (E, 2) (H, 2) (I, 2) (J, 2) (O, 2) (Q, 2) (R, 2) (W, 2) (Y, 2) (Z, 2) (AA, 2) (AB, 2) (AC, 2) and total is 15.
     30       2384       3549       85  -23     388    4     39        n/a       n/a     n/a     n/a          6%
-- Fixed for [O: 1] at level 0.75.
-- Members remaining: (B, 2) (D, 2) (E, 2) (H, 2) (I, 2) (J, 2) (O, 2) (Q, 2) (R, 2) (W, 2) (Y, 2) (Z, 2) (AA, 2) (AB, 2) (AC, 2) and total is 15.
     31       2384       3549       91    6     388    6      6        n/a       n/a     n/a     n/a          5%
-- Fixing column 778 to 1 for member H, was 0.5
-- Members remaining: (B, 2) (D, 2) (E, 2) (H, 2) (I, 2) (J, 2) (O, 2) (Q, 2) (R, 2) (W, 2) (Y, 2) (Z, 2) (AA, 2) (AB, 2) (AC, 2) and total is 15.
     32     2434.5       3549       92    1     395    7      1        n/a       n/a     n/a     n/a          3%
     33     2434.5       3549       86   -6     401    6      0        n/a       n/a     n/a     n/a          4%
-- Fixed for [H: 1] at level 0.75.
-- Members remaining: (B, 2) (D, 2) (E, 2) (I, 2) (J, 2) (O, 2) (Q, 2) (R, 2) (W, 2) (Y, 2) (Z, 2) (AA, 2) (AB, 2) (AC, 2) and total is 14.
     34     2434.5       3549      101   15     404    5      2        n/a       n/a     n/a     n/a          1%
-- Fixing column 1062 to 1 for member I, was 0.5
-- Members remaining: (B, 2) (D, 2) (E, 2) (I, 2) (J, 2) (O, 2) (Q, 2) (R, 2) (W, 2) (Y, 2) (Z, 2) (AA, 2) (AB, 2) (AC, 2) and total is 14.
     35       2837       2837       87  -14     408    4      1        n/a       n/a     n/a     n/a          0%
     36       2736       2736       96    9     415    7      0        n/a       n/a     n/a     n/a          0%
     37       2736       2736       77  -19     419    4      0        n/a       n/a     n/a     n/a          0%
-- Stopping. Objective change is 0 with an integer feasible solution.
Colgen done.
Elapsed time : n/a.
Solution objectives: [2734, 2736].
