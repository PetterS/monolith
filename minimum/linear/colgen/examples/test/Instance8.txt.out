  A:  . . .D=D=N=N= . .D=D=D=N=N= . .D=D=D= . .D=D=D=D=D= . .
  B: E=E=E=N= . . .N=N= . .E=E=E=E=N= . .E=E=E=E= . .E=E= . .
  C: D=D=N=N=N= . .D=D=D= . .D=D=D=D= . . . . . .D=D=D=D=D= .
  D: D=L=L=L=L= . .L=L= . .D=L=L=L=L= . .D=D=L= . .L=L=L= . .
  E: E=E= . .E=D=D=D= . .E=D= . . .E=E=D=N= . . .E=E=N=N=N= .
  F: E=E=E=E=E= . .E=E=E= . .E=E=E= . .N=N=N=N= . .E=E=E= . .
  G: D=D=D=D=N= . .D=D=D=D=D= . . .D=D= . . .D=D=N= . .D=N=N=
  H: E=E=E=D=D= . .E=E=E=E= . . .E=D= . .E=E=E=D=D= . . .E=E=
  I: N=N=N= . .E=N= . .E=E=E=D= . .E=E=E=E= . .E=E=E=E=E= . .
  J: L=N= . . .L=L=L=L=L= . . . .D=D=L=L=L= . .N=N=N= . .L=L=
  K:  .N=N=N=N= . .E=D=D= . .E=E= . .E=D=D= . .E=E=D= . .E=E=
  L: E=E= . .E=E=E=E= . .E=E= . . .E=E=D=D= . . .E=N=N=N=N= .
  M:  .D=D=L=L= . .L=L=L=L=L= . .D=L=L= . .D=L=N=N= . . .N=N=
  N: L=L=L=L=L= . . . . .L=N=N=N=N= . .E=E=E=E=D= . .L=L= . .
  O:  .E=E=E=L= . .L=L=L=L=L= . . .L=L= . .D=L=L=L= . .L=L=L=
  P:  . .E=E=E=E=E= . .E=E=E= . .D=D=D=D=D= . .E=D=D= . .E=E=
  Q: L=L= . . .L=L=N=N=N=N= . . .L=L=L=L= . . .L=L=L=L=L=L= .
  R: D=D=D=D= . . . .D=D=D=D= . . . .D=D=D=D=D=D= . . .D=D=D=
  S: L=L=L=L= . . .D=D=L=L=L= . . . .L=L=L=L=L=L= . . .D=D= .
  T: L=L=L=L=L= . . . .L=L=L=L=L=L= . . .E=D=D=L=L=L= . . . .
  U: D=D=D=D=D= . . .E=E=E=E=E=E= . . .E=E=E=E= . . .N=N= . .
  V:  . . .D=L=L=L=L=L= . . .L=L=L=L=N=N= . . .E=L=L=L=N= . .
  W:  . . .E=E=E=E=N=N= . . . . .E=E=D=N=N= . . .E=E=E=E=E=D=
  X:  . . . . .D=D= . .D= . .D=D=D=D=N= . .D=N= . . . . . . .
  Y: L= . .L= . . . . .D= . .D=D= . . .L=L=L=L= . . .L= . . .
  Z: N= . . . .E=E= . .L= . .E=E=E= . . . . .E=N= . .L= . . .
 AA: L= . . . .L=L=L= . . . .L=L=N=N= . . .L=L= . . . . . . .
 AB:  . . .E= . . . . . . . .D=D=L=N=N= . .L=L= . .L= . .E= .
 AC:  . . . . .D=D= . .N= . . . . . . . . .D=D= .D= . . .D= .
 AD:  . . .L= .D=L= . . . . . . . . . .L= .L=N= . . . . .L= .
Final solution (checked for feasibility): 2734
-- Cover costs  : 2507
-- Roster costs : 227
  A:  . . .D=D=N=N= . .D=D=D=N=N= . .D=D=D= . .D=D=D=D=D= . .
  B: E=E=E=N= . .E=N=N= . .E=E=E=E= . .E=E= . .E=D=D=D=D= . .
  C: D=N=N=N=N= . .D=D=D= . .D=D=D=D= . . . . . .D=D=D=D=D= .
  D: L=L=L=L=L= . .L=L= . .D=L=L=L= . . .D=D=L= . .L=L=L= . .
  E: D=D= . .E=D=D=D= . .E=E= . . .E=E=D=N= . .E=E=N= . .N=N=
  F: E=E=E=E=E= . .E=E=E= . .E=E=E= . .N=N=N=N= . .E=E=E= . .
  G: D=D=D=D=N= . .D=D=D=D=D= . . .D=D= . .D=D=D=N= . . .N=N=
  H: E=E=E=D=D= . .E=E=E=E= . . .E=D= . .E=E=E=D=D= . . .E=E=
  I:  .E=E=E=E= . .E=E=E=E=D= . . .N=N= . .E=E=E=E=E= . .E=D=
  J: L=L=N= . .L=L=L=L=L= . . . .D=L=L=L=L= . .N=N=N= . .L=L=
  K: E=N=N=N=N= . .E=D=D= . .E=E= . .E=D=D= . .E=E=D= . .E=E=
  L: E=E= . .E=E=E=E= . .E=E= . . .E=E=D=D= . . .E=N=N=N=N= .
  M:  .D=D=L=L= . .L=L=L=L=L= . .D=L=N= . .D=L=N=N= . .L=N= .
  N: L=L=L=L=L= . . . . .L=N=N=N=N= . .E=E=E=E=D= . .L=L= . .
  O: E=E=E=L=L= . . .E=L=L=L=L= . .L=L=L= . .E=L=L=L= . . . .
  P:  . .E=E=E=E=E= . .E=E=E= . .D=D=D=D=D= . .D=D=D= . .E=E=
  Q: L=N= . . .L=L=L=L=N=N= . . .E=L=L=L= . . .L=L=L=L=L=L= .
  R: D=D=D=D= . . . .D=D=D=D= . . .D=D=D=D=D= . . .D=D=D=D= .
  S: L=L=L=L= . . .D=D=L=L=L= . . . .L=L=L=L=L=L= . . .D=D=L=
  T: L=L=L=L=L= . . . .L=L=L=L=L=L= . . .E=D=D=L=L=L= . . . .
  U: D=D=D=D=D= . . .E=E=E=E=E=E= . . .E=E=E=E= . . .N=N= . .
  V:  . . .D=L=L=L=L=L= . . .L=L=L=L=N=N= . . .E=L=L=L=N= . .
  W:  . . .E=E=E=E=N=N= . . . . .E=E=D=N=N= . . .E=E=E=E=D=D=
  X:  . . . . .D=D= . .D= . .D=D=D=N= . . .D=D= . . .N= . . .
  Y:  . . .L= . . . . .D= . .D=L= . .L=L=L=L=L= . . . . .L= .
  Z: N= . . . .E=E= . .L= . .E=E=L= . . . . .L= . . .L=N= . .
 AA: L= . . . .L=L=L= . . . .L=L=N=N= . . .L=L= . . . . . . .
 AB:  . . .E= . . . . . . . .D=D=L=N= . .E=L=L=N= . . . .E= .
 AC: N= . . . .D=D= . . . . .D=D= . . . . .D=D= . . . . . . .
 AD:  . . .L= .D=L= . . . . . . . . . . . .L=N= . . .L= .L= .
Final solution (checked for feasibility): 2736
-- Cover costs  : 2509
-- Roster costs : 227
